.gitignore

```diff
@@ -2,6 +2,7 @@
 *.retry
 pontus.larsson_medsci.uu.se.key
 charon_credentials.yml
-statusdb_creds.yml
+statusdb_creds_prod.yml
```

Comment:
After discussion we concluded that this is fine.
But yes, it needs a status_db credentials file.